# This message contains information about the position of n colored cups
uint32 		n_cups 		#number of cups recorded in the workspace
string[] 	cups_color 	#vector with the designations given to each cup
float32[] 	cups_pos_x 	#this is a n matrix with the x position of each cup (in millimeters)
float32[] 	cups_pos_y 	#this is a n matrix with the y position of each cup (in millimeters)
float32[] 	cups_pos_z 	#this is a n matrix with the z position of each cup (in millimeters)